### ISSUE TITLE
How to get the list of client_id for an experiment/alternative ?

### ISSUE THREAD

**Issue by hugocar** [seatgeek/sixpack#257]
Thanks for SixPack
What is the recommended way to **get the users associated to an alternative**  ?

- One way is to use participate but it forces participation and looping over users.

- An other way is to get the client_id list from redis 'sip:e:experiment:users' and loop participate over. But it doesn't seem optimal.

What would you recommand?

What happened to https://github.com/seatgeek/sixpack/pull/112 ? seemed like a great idea
Can something like `/experiments/<experiment_name>/alternatives/<alternative_name>/clients` ever be implemented ? (would you accept PRs going this direction?)

**Comment by Singwai** [comment]
@zackkitzmiller I also agree this is a good suggestion. Will you merge it if we have it implement.
